wazuh/wazuh-kibana-app
# [Fix] Remove URL query param `redirectRule` on CDB Lists/Decoders

### Description
Remove unnecessary URL param in `CDB Lists/Decoders` when clink to expand the details about the list/decoder.

This PR removes from the URL:
```
redirectRule=undefined
```

### Testing
For `CDB Lists` and `Decoders`:
1. Go to `section`
2. Click in a table row to expand the details and review the URL that doesn't contain the `redirectRule` query param.

- For the `Rules` section the query param should appear when clicking on the table row to expand the details.

Closes: #3356
